whois-update/src/main/java/net/ripe/db/whois/update/dns/zonemaster/domain/StartDomainTestRequest.java

```diff
@@ -68,6 +71,10 @@ private List<StartDomainTestRequest.Nameserver> parseNameservers(final Set<CIStr
         return nameservers;
     }
 
+    private String removeDomainLastDot(String domain) {
```

Comment:
Suggestion :rename to removeTrailingDot? or something similar it is more widely accepted term.